app/scripts/views/settings/avatar_camera.js

```diff
@@ -71,11 +72,6 @@ function (_, Cocktail, canvasToBlob, FormView, ProgressIndicator,
                              nav.mozGetUserMedia ||
                              nav.msGetUserMedia;
 
-      if (! getUserMedia) {
```

Comment:
Since the existence of getUserMedia is checked in beforeRender, it no longer needs to be checked here.